### PR TITLE
[15] Research Programming Language

### PR DESCRIPTION
**Pull request by Jeffbuchi**
Did a research on programming languages.

### CODE REVIEW

**Review by adaugochi — CHANGES_REQUESTED** [review]
I just requested for access to your doc

File: submissions/jeffersonMormah/css/contact.css [hunk 1]
```diff
@@ -226,6 +230,7 @@ main {
     text-align: center;
 }
 .footer1, .footer2, .footer3 {
+    width: 44%;
```
Comment (ToluAkin) [inline submissions/jeffersonMormah/css/contact.css:233]: You are declaring this twice??? @Jeffbuchi

File: submissions/jeffersonMormah/css/contact.css [hunk 2]
```diff
@@ -218,6 +221,7 @@ footer p{
         float: none;
 }
 main, aside {
+    width: 70%;
```
Comment (ToluAkin) [inline submissions/jeffersonMormah/css/contact.css:224]: You are declaring this twice??? @Jeffbuchi

File: submissions/jeffersonMormah/css/contact.css [hunk 3]
```diff
@@ -174,11 +175,13 @@ footer {
 }
 .footer2 input[type="email"]{
    height: 25px;
+   padding: 3px 18px 3px ;
```
Comment (ToluAkin) [inline submissions/jeffersonMormah/css/contact.css:178]: Remove this @Jeffbuchi

**Review by ToluAkin — CHANGES_REQUESTED** [review]

File: submissions/jeffersonMormah/css/contact.css (hunk 1 above)
Comment (Jeffbuchi) [reply submissions/jeffersonMormah/css/contact.css]: Rectified

File: submissions/jeffersonMormah/css/contact.css (hunk 2 above)
Comment (Jeffbuchi) [reply submissions/jeffersonMormah/css/contact.css]: Rectified

**Comment by Jeffbuchi** [comment]
> I just requested for access to your doc

Just Granted access @adaugochi @ToluAkin

File: submissions/jeffersonMormah/css/contact.css (hunk 3 above)
Comment (Jeffbuchi) [reply submissions/jeffersonMormah/css/contact.css]: Rectified

**Pull request merged**